CODE_OF_CONDUCT.md

```diff
@@ -7,8 +7,8 @@ This project has adopted the [OCaml Code of Conduct](https://github.com/ocaml/co
 This project follows the OCaml Code of Conduct [enforcement policy](https://github.com/ocaml/code-of-conduct/blob/main/CODE_OF_CONDUCT.md#enforcement).
 
 To report any violations, please contact:
-Thomas Gazagnaire <thomas [at] tarides [dot] com>
-Arthur Wendling <arthur [at] tarides [dot] com>
-Kevin Smith <kevin [at] tarides [dot] com>
-Gwenaëlle Lecat <gwenaelle [at] tarides [dot] com>
-Vincent Balat <vincent.balat [at] tarides [dot] com>
+- Thomas Gazagnaire <thomas [at] tarides [dot] com>
+- Arthur Wendling <arthur [at] tarides [dot] com>
+- Kevin Smith <kevin [at] tarides [dot] com>
```

Comment:
```suggestion
```
Kevin is unlikely to be available for Irmin's code of conduct violations (and I don't know his latest email address)